integration-test/src/main/java/com/sequenceiq/it/cloudbreak/ResourceGroupTest.java

```diff
@@ -5,10 +5,17 @@
 
     public static final String AZURE_RESOURCE_GROUP_USAGE_SINGLE = "SINGLE";
 
+    public static final String AZURE_RESOURCE_GROUP_USAGE_SINGLE_DEDICATED = "SINGLE_WITH_DEDICATED_STORAGE_ACCOUNT";
+
     public static final String AZURE_RESOURCE_GROUP_USAGE_MULTIPLE = "MULTIPLE";
 
     public static final String AZURE_RESOURCE_GROUP_NAME = "AZURE_RESOURCE_GROUP_NAME";
 
     private ResourceGroupTest() {
```

Comment:
Usually we `extends GherkinTest` at this level of the project. If you do not judge, this is not worth it, we can leave this at it is.